src/middlewared/middlewared/api/v25_04_0/docker_network.py

```diff
@@ -0,0 +1,19 @@
+from pydantic import ConfigDict
+
+from middlewared.api.base import BaseModel, NonEmptyString
+
+
+class DockerNetworkEntry(BaseModel):
+
+    model_config = ConfigDict(
```

Comment:
What extra fields are there?

Reply:
Docker api provides a bunch of fields we are not really concerned about atm and hence extra has been allowed. If you think we should be restrictive, i can do that ?

Reply:
Thing is if we don't remove those fields, someone might start using them. Then the fields might disappear/change without notice, and things will break. My opinion is that we should either document those fields explicitly or remove them.